app.go

```diff
@@ -151,6 +156,10 @@ func (a *App) Setup() {
 		a.Action = helpCommand.Action
 	}
 
+	if a.InterruptHandler == nil {
+		a.InterruptHandler = CancelContextOnInterrupt
+	}
```

Comment:
Should not do this, user might set it to null not to get the context cancelled.

Reply:
Essentially, if this is null, we should not even try to setup signal handling, and leave it up to the user.

Reply:
I think if it's set to nil, we should actually set it to the Noop handler. We can add that to the documentation as well. It's easier that way to not cause nil panics